packages/vm/tests/api/utils.ts

```diff
@@ -12,6 +12,13 @@ export function createAccount(nonce: BN = new BN(0), balance: BN = new BN(0xfff3
   return new Account(nonce, balance)
 }
 
+export async function setBalance(vm: VM, address: Address, balance = new BN(100000000)) {
+  const account = createAccount(new BN(0), balance)
+  await vm.stateManager.checkpoint()
+  await vm.stateManager.putAccount(address, account)
+  await vm.stateManager.commit()
```

Comment:
Out of interest: what happens without this outer `checkpoint()` and `commit()` wrapping? Or is this just to play safe (does this add anything here on this?)?